### DISCUSSION TITLE
Error Message in Form with Steps

### DISCUSSION THREAD

**Discussion by PGLange** [Budibase/budibase#2916] [Help] [answered]
Hi all,
pls give me a hint how to open error messanges from Budibase in the docker containers, the one shown in the platform just yelds "An error has occured". I hope I could get some more details in the logs...

Thanks

**Comment by mjashanks** [comment]
If you really need to get into the docker containers, you will need to use `docker exec`:

- https://docs.docker.com/engine/reference/commandline/exec/
- https://stackoverflow.com/questions/30172605/how-do-i-get-into-a-docker-containers-shell

However, first check your dev tools console - see if there are any console error messages or failed network requests. That should give some more information. Feel free to post any error messages back here.

**Comment by PGLange** [comment, marked as answer]
Hi @mjashanks,
My Budibase is version 0.9.136. With firefox networks analysis tool, I get this MSG:
![error](https://user-images.githubusercontent.com/86222990/136373277-f9be3388-4ca7-4bc3-83eb-1aa8aeb6187a.PNG)

**Reply by mjashanks** [reply]
Hmm - would you be able to email me an export of your app?

Also - do you get any more info in the "Response" tab for this network request?

**Reply by mjashanks** [reply]
Looking at your export. I can reproduce your issue, but in a strange way...

If i click save on the screen below, I get your errror. However, this makes sense to me - you are sending a blank object:

![image](https://user-images.githubusercontent.com/3524181/136534954-7dee1564-0061-445f-98d9-d4a05da24512.png)

So, if I then create the form by
1. going to the `/dfsa/new/row` screen
2. click on the `Field Group` component
3. Click on `Update Form Fields`, in the settings panel
4. Re-run my app, and access the screen again, at `/#/dsfa/new/row`
5. Fill out the form, with valid values, and click `Save`

Then, the form saves for me ok.
